MonoDevelop.MSBuild/Language/MSBuildDocumentValidator.cs

```diff
@@ -679,7 +679,7 @@ void VisitPureLiteral (MSBuildElementSyntax elementSymbol, MSBuildAttributeSynta
 			case MSBuildValueKind.NuGetVersion:
 			case MSBuildValueKind.VersionSuffixed:
 				// TODO
-				if (!NuGet.Versioning.NuGetVersion.TryParse (value, out _)) {
+				if (!NuGet.Versioning.VersionRange.TryParse (value, out _)) {
```

Comment:
This will lead to incorrect validation for properties of type `version-suffixed`. We need to separate the switch cases for `NuGetVersion` and `VersionSuffixed` so that `VersionSuffixed` can continue to use `NuGetVersion.TryParse()`